Author: Rhonda D'Vine <rhonda@debian.org>
Description: Remove --force option from po4a calls
  The --force option forced a regenerate of the pot files on every call,
  hindering the reproducible buildability of the package

Index: VCS/GNUmakefile
===================================================================
--- VCS.orig/GNUmakefile	2015-12-03 17:18:38.980045908 +0100
+++ VCS/GNUmakefile	2016-01-20 13:22:32.795072663 +0100
@@ -173,12 +173,12 @@
 # create (or update) dctrl-tools-man.pot
 man/po4a/po/dctrl-tools-man.pot :
 	touch man/po4a/po/dctrl-tools-man.pot
-	$(PO4A) --force --no-translations $(PO4A_CONFIG)
+	$(PO4A) --no-translations $(PO4A_CONFIG)
 
 # build translated manpages in man/translated/$lang/
 po4a :
 	touch man/po4a/po/dctrl-tools-man.pot
-	$(PO4A) --force --no-backups $(PO4A_CONFIG)
+	$(PO4A) --no-backups $(PO4A_CONFIG)
 
 # handle the "%.1 : %.1.cp" rule if and only if the grep-dctrl.1.cp file exists
 translated-man : po4a
